### PR TITLE
We had a wrong format in the long datetime option

### PR DESCRIPTION
**Pull request by lagartoverde**
Small bug on the default formats that caused some problems on qa

### CODE REVIEW

File: addon/plugins/rdfa-date-plugin/index.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ export const defaultDateFormats: DateFormat[] = [
     label: 'Long Date',
     key: 'long',
     dateFormat: 'EEEE dd MMMM yyyy',
-    dateTimeFormat: 'EEEE dd MMMM yyyy at HH:mm',
+    dateTimeFormat: 'EEEE dd MMMM yyyy \'at\' HH:mm',
```
Comment (abeforgit) [inline addon/plugins/rdfa-date-plugin/index.ts:14]: we should use `PPPPp` instead, now there's a random `at` that's not localized

**Review by abeforgit — CHANGES_REQUESTED** [review]

File: addon/plugins/rdfa-date-plugin/index.ts (hunk 1 above)
Comment (lagartoverde) [reply addon/plugins/rdfa-date-plugin/index.ts:14]: true, sorry you told me, it's monday, the problem with PPPPp is that it doesn't give the user a start from where customize it's date, but I think is the best solution

Comment (abeforgit) [reply addon/plugins/rdfa-date-plugin/index.ts]: what do you mean with "us date"?

Comment (lagartoverde) [reply addon/plugins/rdfa-date-plugin/index.ts]: Like from DD/MM/YYYY hh:mm, the user can easily change it to replace the separators, add new words...., this doesn't happen with the PPPPp but I think is also okay

**Review by elpoelma — APPROVED** [review]

**Pull request merged**